nsxt/resource_nsxt_policy_tier1_gateway_interface.go

```diff
@@ -316,14 +317,21 @@ func resourceNsxtPolicyTier1GatewayInterfaceDelete(d *schema.ResourceData, m int
 func resourceNsxtPolicyTier1GatewayInterfaceImport(d *schema.ResourceData, m interface{}) ([]*schema.ResourceData, error) {
 	importID := d.Id()
 	s := strings.Split(importID, "/")
```

Comment:
cosmetics: lets move this below the path import section?

Reply:
The split? It's used at line 323 to get the locale_service_id and then again in 328. So IDK if there's a better place for it.
Unless you mean something else.

Reply:
Correct, my bad, sorry. But it seems wrong to reference s[9] below? Doesn't index depend on whether tenant is present in path? And the user can provide path with smaller amount of tokens, that would still be policy path

Reply:
Indeed. It's better to locate "locale_service" and grab the next list member.